content/basics/ably.textile

```diff
@@ -0,0 +1,89 @@
+---
+title: What is Ably
+meta_description: "A brief introduction to Ably and its features."
+---
+
+Ably’s realtime platform enables data to be streamed between Internet connected systems in milliseconds through our Realtime or REST interfaces. Built on the "Four Pillars of Dependability":#infrastructure Ably provides reliable performance available when needed.
+
+h2(#what). What can you build with Ably?
+
+Ably is used across a variety of industries to provide scalable:
+
+* Live updates
+* Chat applications
+* Virtual events
+* Multi-user virtual spaces and applications
+* And much more
+
+Below we will cover some of the different APIs and features Ably provides which will give you a deeper insight into what can be done with Ably. There's also the "How to use Ably page":/basics/use-ably which will go over the basics of connecting with Ably.
+
+Alternatively take a look at our "Use cases and solutions":https://ably.com/solutions page for specific example use cases.
+
+h2(#about). About the APIs
+
+Ably supports both realtime and REST interfaces. The "Realtime library":/basics/use-ably#realtime-interface establishes a WebSocket connection allowing realtime messages to be efficently sent both to and from the Ably. The "REST library":/basics/use-ably#rest-interface communicates using the HTTP protocol for all operations and is effectively stateless.
+
+There are a number of "Ably SDKs":https://ably.com/download in different languages to match your needs.
+
+Some of the key features are: 
+
+* "PubSub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). PubSub with Channels
+
+Ably uses the Publish/Subscribe (Pub/Sub) architectural pattern to allow any number of publishers to publish data and any number of subscribers to subscribe to a source of data. This works hand in hand with Ably's concept of "channels". Channels are used to aggregate all data into a single location, allowing publishers to publish to just one location and subscribers to subscribe to just one location.
+
+Read more in our "Pub/Sub section":/channels/.
+
+h3(#presence-and-occupancy). Presence and Occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel). You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
```

Comment:
Avoid use of brackets. Move out to new sentence along the lines of: "Examples of presence events include..."

"You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API."

If I'm using the client library SDK would I need to do that?